pom.xml

```diff
@@ -102,9 +107,9 @@
     </build>
 
     <scm>
-        <url>https://github.com/batkinson/Maven-Pack200-Plugin</url>
-        <connection>scm:git:https://github.com/batkinson/Maven-Pack200-Plugin.git</connection>
-        <developerConnection>scm:git:git@github.com:batkinson/Maven-Pack200-Plugin.git</developerConnection>
+        <url>https://github.com/Pixeon/Maven-Pack200-Plugin</url>
+        <connection>scm:git:https://github.com/Pixeon/Maven-Pack200-Plugin.git</connection>
+        <developerConnection>scm:git:git@github.com:Pixeon/Maven-Pack200-Plugin.git</developerConnection>

```

Comment:
Same as above. This seems like you meant to have your own release and forgot that it would update the pr.